src/integration/pages/storyPage/crossPlatformTests.js

```diff
@@ -12,8 +12,65 @@ export default () => {
     expect(imageEl).toBeInTheDocument();
     expect(imageCaptionEl).toBeInTheDocument();
     expect(imageCaptionEl.textContent).toBeTruthy();
-    expect(imageCaptionEl.textContent).toMatchInlineSnapshot(
-      `"Pie de foto, Llegó el día de la salida de Reino Unido de la UE."`,
-    );
+    expect(imageCaptionEl.textContent).toMatchSnapshot();
+  });
+
+  describe('Social Embeds', () => {
+    it('I can see the skip link', () => {
+      const skipLinkEl = document.querySelector(
+        'a[href="#skip-youtube-content-1"]',
+      );
+
+      expect(skipLinkEl).toBeInTheDocument();
+      expect(skipLinkEl.textContent).toBeTruthy();
+      expect(skipLinkEl.textContent).toMatchSnapshot();
+    });
+
+    it('I can see the skip link destination', () => {
+      const skipLinkDestinationEl = document.getElementById(
+        'skip-youtube-content-1',
+      );
+
+      expect(skipLinkDestinationEl).toBeInTheDocument();
+      expect(skipLinkDestinationEl.textContent).toBeTruthy();
+      expect(skipLinkDestinationEl.textContent).toMatchSnapshot();
+    });
+
+    /**
+     * Rich embeds.
+     */
+    const hasRichYouTubeEmbed = [
+      document.querySelector('iframe[src*="https://www.youtube.com/embed/"]'),
+      document.querySelector('amp-youtube'),
+    ].find(el => !!el);
+
+    if (hasRichYouTubeEmbed) {
+      it('I can see the caption accompanying a YouTube embed', () => {
+        const figCaptionEl = document.querySelector(
+          'a[href="#skip-youtube-content-1"] + figure > figcaption',
+        );
+
+        expect(figCaptionEl).toBeInTheDocument();
+        expect(figCaptionEl.textContent).toBeTruthy();
+        expect(figCaptionEl.textContent).toMatchSnapshot();
+      });
+    }
+
+    /**
+     * Fallbacks.
+     */
+    const hasFallback = !!document.getElementById('skip-facebook-content-1');
+
+    if (hasFallback) {
+      it('I can see a link to the original content in a fallback', () => {
+        const externalLinkEl = document.querySelector(
+          'a[href*="https://www.facebook.com"]',
+        );
+
+        expect(externalLinkEl).toBeInTheDocument();
+        expect(externalLinkEl.textContent).toBeTruthy();
```

Comment:
I have found that this is a neat way of  including URL & text content in a single snapshot:

```
expect(externalLinkEl.getAttribute('href)).toMatchSnapshot(externalLinkEl.textContent);
```

Adding a string as a param for `toMatchSnapshot` adds it to the snapshot description. 

Just a thought - don't feel obliged to do this.

Reply:
Thanks, Karina. I will remember that for future tests. This one doesn't need to snapshot `href`, because it's allowed to change – `textContent` is static however.